build.gradle

```diff
@@ -0,0 +1,71 @@
+buildscript {
+    repositories {
+        maven { url 'https://plugins.gradle.org/m2' }
+    }
+
+    dependencies {
+        //classpath 'gradle.plugin.net.minecrell:licenser:0.4.1'
+		classpath "io.freefair.gradle:lombok-plugin:3.6.4"
+    }
+}
+
+defaultTasks 'licenseFormat', 'build'
+
+subprojects {
+    apply plugin: 'java'
+    apply plugin: 'maven'
+    //apply plugin: 'net.minecrell.licenser'
+	//apply plugin: "io.freefair.lombok"
+
+    group = 'me.confuser.banmanager'
+    version = '6.0.1-SNAPSHOT'
```

Comment:
Can we set this to `7.0.0-SNAPSHOT` please? They'll be some major breaking changes with this which will break compatibility with other plugins.